api.md

```diff
@@ -170,6 +172,61 @@ SELECT add_dimension('conditions', 'device_id', number_partitions => 2);
 SELECT add_dimension('conditions', 'device_id', number_partitions => 2, if_not_exists => true);
 ```
 
+---
+## attach_data_node() [](attach_data_node)
+
+Attach a data node to a hypertable. The data node should have been
+previously created using [`add_data_node`](#add_data_node).
+
+When a distributed hypertable is created it will by default use all
+available data nodes for the hypertable, but if a data node is added
+*after* a hypertable is created, the data node will not automatically
+be used by existing distributed hypertables.
+
+If you want a hypertable to use a data node that was created later,
+you must attach the data node to the hypertable using this
+function.
+
+#### Required Arguments [](attach_data_node-required-arguments)
+
+| Name              | Description                                   |
+|-------------------|-----------------------------------------------|
+| `node_name`       | Name of data node to attach             |
+| `hypertable`      | Name of hypertable to attach node to          |
+
+#### Optional Arguments [](attach_data_node-optional-arguments)
+
+| Name              | Description                                   |
+|-------------------|-----------------------------------------------|
+| `if_not_attached` | Prevents error if data node is already attached to the hypertable. A notice will be printed that the data node is attached. |
+
+#### Returns
+
+| Column               | Description                              |
+|-------------------|-----------------------------------------------|
+| `hypertable_id`      | Hypertable id of the modified hypertable |
+| `node_hypertable_id` | Hypertable id on the remote data node    |
+| `node_name`          | Name of the attached data node     |
+
+#### Sample Usage [](attach_data_node-examples)
+
+Attach a data node `dn3` to a distributed hypertable `conditions`
+previously created with
+[`create_distributed_hypertable`](#create_distributed_hypertable).
+
+```sql
+SELECT * FROM attach_data_node('dn3','conditions');
+
+hypertable_id | node_hypertable_id |  node_name  
+---------------+--------------------+-------------
+            5 |                  3 | dn3
+
+(1 row)
+```
+
+>:TIP: You must add a data node to your distributed database first
```

Comment:
```suggestion
>:TIP: You must add a data node to your distributed database with [`add_data_node`](#add_data_node) first
```

Reply:
I assume there is no other way to add a data node?

Reply:
Nope! That's the only way to do it. Will add.

Reply:
Done